apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -260,6 +262,18 @@ class ScriptEditor extends React.Component {
       is_migrated: this.props.isMigrated
     };
 
+    if (this.state.weeklyInstructionalMinutes) {
```

Comment:
I'm wondering if we want to check that weeklyInstructionalMinutes is great than zero here